openshift/4.0/templates/database/mssql-deploy.yaml

```diff
@@ -91,6 +190,9 @@ objects:
             - name: mssql-data
               persistentVolumeClaim:
                 claimName: ${PVC_NAME}${INSTANCE}
+            # - name: backup
```

Comment:
Same here

Reply:
Nope, I had originally removed it thinking the same thing.  But it was added intentionally so that it is easier to apply for database backup.